docs/examples/kube2iam.rst

```diff
@@ -0,0 +1,242 @@
+Kube2IAM
+--------
+
+Kube2IAM is an extension to kubernetes that will allow you to give
+fine grained AWS access to pods without. More information about the
+project can be found on the `project page <https://github.com/jtblin/kube2iam>`_.
+
+Prerequisite
+~~~~~~~~~~~~
+
+Make sure `HELM <https://www.helm.sh/>`_ is `activated <https://docs.tarmak.io/user-guide.html#tiller>` on the Tarmak cluster.
```

Comment:
link isn't displaying properly

Reply:
Possibly refer to the tarmak docs for installing tiller into the cluster too rather than suggesting using `helm init`

Reply:
@dippynark link is fixed and it was already linked to the docs of tarmak to setup helm.